plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -312,6 +332,21 @@ export const createPublishGithubPullRequestAction = (
       );
 
       try {
+        const gitAuthorInfo = {
+          name: gitAuthorName
+            ? gitAuthorName
+            : config.getOptionalString('scaffolder.defaultAuthor.name'),
+          email: gitAuthorEmail
+            ? gitAuthorEmail
+            : config.getOptionalString('scaffolder.defaultAuthor.email'),
+        };
+
+        // use provided info if possible, otherwise use fallbacks
+        const authorInfo = {
```

Comment:
I wonder if we can use the token that we're going to use here to set some sane defaults. Providing we can get name and email from the token and the `/user` endpoint?

Reply:
Hi, i was inspired by github publish action, where the defaults are set https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts#L163

Reply:
Yea - I think there's also room to update that there too. My problem with this change is that we didn't provide a `committer` before, what is the commiter set as? Is it the signed in users token details? In that case then we probably want to keep that backwards compatibility by not adding one instead?

Reply:
@legendik just a ping here ^